pages/lib/global_scheme.py

```diff
@@ -1,62 +1,58 @@
 import plotly.io as pio
-
-from pages.lib.global_variables import Variables, VariableInfo
+import numpy as np
+from pages.lib.global_variables import Variables, VariableInfo, CET_L19_10
```

Comment:
remove the color palette import and define in here the color palette as list of colors